docs/guides/index.mdx

```diff
@@ -13,6 +13,9 @@ In addition, many open-source projects are part of the broader Qiskit ecosystem.
 
 ![All Qiskit pattern steps are shown (Map problem, Optimize for hardware, Execute on hardware, and Post-process results). All steps, except for 'Execute on hardware', use the Qiskit SDK. 'Optimizing for hardware' additionally uses the Qiskit Transpiler Service.  'Executing on hardware' uses only the Qiskit Runtime Service.](/images/qiskit-patterns/patterns.svg)
 
+<Admonition type="note" title="Responsible quantum computing">
+IBM is committed to the responsible development of quantum computing. Please familiarize yourself with the terms and conditions, which forbid uses of our services relating to weaponry, defeating encryption, and processing regulated data. Learn more about responsible quantum at IBM, review our responsible quantum principles, and flag responsible quantum computing issues in the [Responsible quantum computing and inclusive tech](./responsible-quantum-computing) topic.
```

Comment:
The link should be `/` without a `.`. Ditto the link in `hello-world`

```suggestion
IBM is committed to the responsible development of quantum computing. Please familiarize yourself with the terms and conditions, which forbid uses of our services relating to weaponry, defeating encryption, and processing regulated data. Learn more about responsible quantum at IBM, review our responsible quantum principles, and flag responsible quantum computing issues in the [Responsible quantum computing and inclusive tech](/responsible-quantum-computing) topic.
```

Reply:
I'm figuring it out :) Still in draft